src/main/java/frc/robot/utils/MoSparkMaxPID.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.utils;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMax.ControlType;
+
+public class MoSparkMaxPID {
```

Comment:
Overall, this looks good!

One suggestion for improvement: have MoSparkMaxPID implement the `GraphValues` interface. Then you don't need as many lambdas in `TunerUtils.forSparkMaxSmartMotion()`.

You could replace:
```java
.buildGraphValues()
.withSetpointFrom(sparkMax::getReference)
.withLastMeasurementFrom(sparkMax::getMeasurement)
.withLastOutputFrom(sparkMax::getOutput)
.finishGraphValues()
```
with
```java
.withGraphValues(sparkMax)
```

which, IMO, is a bit cleaner.